public/app/core/services/echo/Echo.ts

```diff
@@ -82,7 +82,7 @@ export class Echo implements EchoSrv {
         height: window.screen.height,
       },
       userAgent: window.navigator.userAgent,
-      ts: performance.now(),
+      ts: new Date().getTime(),
```

Comment:
the timestamp for the event, changed this to normal ms timestamp, not sure we need this to be using high res timer, as I do not think we are going to measure the time difference between two event reports in highres. Otherwise consumers have to remember to convert this before saving/using it in api call. 

@dprokop thoughts?

Reply:
`performance.now()` has a benefit of being identical to the lifespan of a single tab, so we know how long since opening the tab did it take for the user to perform an action. I would keep the the as performance.now() and add additional information meta information as `datetimeTs` which is exactly what you are doing here.

Reply:
But when do need the lifespan of the tab on the event? I would assume ts field to be a epoch timestamp 

Performance.now is  useful for measuring small durations, so maybe a special field for that if we need to measure sub millisecond difference between report events

Reply:
added timeSinceNavigationStart